core/templates/dev/head/collection_editor/CollectionEditorDataService.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
I think you shouldn't add this file. Instead, use https://github.com/oppia/oppia/blob/221d4a25be12e0a20e5dc46a120a089dd4894079/core/templates/dev/head/domain/collection/CollectionDataService.js. If you end up needing to add additional information to the data service that the editor needs but the learner view doesn't, please ping me so we can discuss it. There's a way it can be done while keeping the data sent to the learner view minimal while simultaneously avoiding a need for an editor-specific data service.